fhttp/http.go

```diff
@@ -48,7 +48,7 @@ var (
 	// Host is treated specially, remember that one separately.
 	hostOverride string
 	// BufferSizeKb size of the buffer (max data) for optimized client in kilobytes defaults to 32k.
-	BufferSizeKb = 32
+	BufferSizeKb = 128
```

Comment:
changed so fortio can proxy itself without error (proxy the shrunk chart.js which is about 122k)